From b37758a55c85c3aacd33e531f906d080853ccec0 Mon Sep 17 00:00:00 2001
From: Isaac Fernandes <101341132+Eyezuhk@users.noreply.github.com>
Date: Mon, 2 Dec 2024 16:37:42 -0300
Subject: [PATCH 10/20] Add .lycheeignore file with linkedin to stop error on
 workflow Linkcheck (#2868)

Add a .lycheeignore file with LinkedIn to stop the error in the workflow
Link Checker due to response 999, and update to the new version
lycheeverse/lychee-action@v2.1.0.
---
 .github/workflows/broken-links.yml | 2 +-
 .lycheeignore                      | 2 ++
 2 files changed, 3 insertions(+), 1 deletion(-)
 create mode 100644 .lycheeignore

diff --git a/.github/workflows/broken-links.yml b/.github/workflows/broken-links.yml
index 3f76faeb..0c35d199 100644
--- a/.github/workflows/broken-links.yml
+++ b/.github/workflows/broken-links.yml
@@ -45,7 +45,7 @@ jobs:
       - uses: actions/checkout@v4
 
       - name: Link Checker ðŸ”—
-        uses: lycheeverse/lychee-action@v1.9.0
+        uses: lycheeverse/lychee-action@v2.1.0
         with:
           fail: true
           # removed md files that include liquid tags
diff --git a/.lycheeignore b/.lycheeignore
new file mode 100644
index 00000000..b736879b
--- /dev/null
+++ b/.lycheeignore
@@ -0,0 +1,2 @@
+linkedin.com
+reddit.com
\ No newline at end of file
-- 
2.48.1

